source/quick-start-legacy.txt

```diff
@@ -0,0 +1,115 @@
+===========================
+Quick Start - Legacy Driver
+===========================
+
+.. default-domain:: mongodb
+
+.. contents:: On this page
+   :local:
+   :backlinks: none
+   :depth: 2
+   :class: singlecol
+
+.. admonition:: This guide is for the legacy Java API only
+   :class: important
+
+   Unless your application needs to use the Legacy Java API, use the
+   :doc:`Quick Start Guide </quick-start>` instead. 
+
+.. include:: /includes/quick-start/overview.rst
+
+
+Set up Your Project
+-------------------
+
+Install the Java Development Kit (JDK)
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+.. include:: /includes/quick-start/install-jdk.rst
+
+Create the Project
+~~~~~~~~~~~~~~~~~~~
+
+.. include:: /includes/quick-start/create-project.rst
+
+Add MongoDB as a Dependency
+~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+If you are using Maven, add the following to your ``pom.xml`` dependencies
+list:
+
+.. code-block:: xml
+
+   <dependencies>
+       <dependency>
+           <groupId>org.mongodb</groupId>
+           <artifactId>mongodb-driver-legacy</artifactId>
+           <version>4.0.1</version>
+       </dependency>
+   </dependencies>
```

Comment:
```suggestion
   </dependencies>
       <properties>
        <maven.compiler.source>1.9</maven.compiler.source>
        <maven.compiler.target>1.9</maven.compiler.target>
    </properties>
```

Reply:
I had to specify this as maven defaults to 1.5, or at least it did on my machine.